clouds/aws/create_aws_network_resources.py

```diff
@@ -0,0 +1,793 @@
+import json
+import os
+from ipaddress import IPv4Network
+
+import click
+from aws_utils import verify_aws_config, verify_aws_credentials
+from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
+from simple_logger.logger import get_logger
+
+from clouds.aws.session_clients import ec2_client as test_client
+
+LOGGER = get_logger(name=__name__)
+
+
+def ec2_assign_name_tag(ec2_client, res_id, tag_name) -> None:
+    """
+    Assign 'Name' tag value to EC2 resource
+
+    Args:
+         ec2_client (botocore.client.EC2): aws botocore client
+         resource_id (str): The ID of resource(which can be of 'vpc', 'subnet', etc)
+         tag_name (str): The value of 'Name' tag
+    """
+    ec2_client.create_tags(
+        Resources=[
+            res_id,
+        ],
+        Tags=[
+            {
+                "Key": "Name",
+                "Value": tag_name,
+            },
+        ],
+    )
+
+
+def get_subnets_cidr(cidr_block, subnet_count) -> list:
+    """
+    Calculates Subnet CIDR address blocks within given VPC CIDR block
+
+    Args:
+         ec2_client (botocore.client.EC2): aws botocore client
+         cidr_block (str): The IPv4 network of VPC
+         subnet_count (str): The number of additional bits with which to extend the prefix
+
+    Returns:
+         list: The list of subnet CIDR address blocks
+
+    Raises:
```

Comment:
this function do not raise anything (IPv4Network will raise and not this function)
please remove it from the docstring.

Reply:
done